### ISSUE TITLE
Is bibdesktoword compatible with word 2016?

### ISSUE THREAD

**Issue by aureincm-zz** [ackman678/bibdesktoword#4]
Hi there, 
I recently upgraded from word 2011 to word 2016, and am having issues using Bibdesktoword (v0.17 running with Bibdesk Version 1.6.7).
Is Bibdesktoword compatible with word 2016? Or should I just go back to word 2011?
Thanks for your guidance (and for this AMAZING software btw!)
 - Aurelie

**Comment by ackman678** [comment]
Yes I recommend sticking with Word 2011 for now as there is a known issue with Word 2015 that still needs to be resolved
